Library/Homebrew/livecheck/strategy.rb

```diff
@@ -161,7 +161,7 @@ def from_url(url, livecheck_strategy: nil, url_provided: false, regex_provided:
             # specifies the strategy and contains a `strategy` block
             next if (livecheck_strategy != strategy_symbol) || !block_provided
           elsif strategy.const_defined?(:PRIORITY) &&
-                !strategy::PRIORITY.positive? &&
+                !strategy.const_get(:PRIORITY).positive? &&
```

Comment:
Sorbet doesn't support [dynamic constant references](https://sorbet.org/docs/error-reference#5001) using the prior format, but i believe this equivalent